yt_dlp/extractor/youtube.py

```diff
@@ -3307,7 +3306,57 @@ def _extract_heatmap(self, data):
                 'value': ('intensityScoreNormalized', {float_or_none}),
             })) or None
 
-    def _extract_comment(self, comment_renderer, parent=None):
+    def _extract_comment(self, view_model, entities, parent=None):
+        comment_entity_payload = traverse_obj(entities, (..., 'payload', 'commentEntityPayload', {dict}), get_all=False)
+        toolbar_entity_payload = traverse_obj(entities, (..., 'payload', 'engagementToolbarStateEntityPayload', {dict}), get_all=False)
+        comment_id = comment_entity_payload.get('properties').get('commentId')
+
+        info = {
+            'id': comment_id,
+            'text': try_get(comment_entity_payload, lambda x: x['properties']['content']['content'], str),
+            'like_count': parse_count(try_get(comment_entity_payload, lambda x: x['toolbar']['likeCountA11y'], str)) or 0,
+            'author_id': traverse_obj(comment_entity_payload, ('author', 'channelId', {self.ucid_or_none})),
+            'author': try_get(comment_entity_payload, lambda x: x['author']['displayName'], str),
+            'author_thumbnail': traverse_obj(comment_entity_payload, ('author', 'avatarThumbnailUrl', {url_or_none})),
+            'parent': parent or 'root',
+        }
+
+        # Timestamp is an estimate calculated from the current time and time_text
+        time_text = try_get(comment_entity_payload, lambda x: x['properties']['publishedTime'], str) or ''
+        timestamp = self._parse_time_text(time_text)
+
+        info.update({
+            # FIXME: non-standard, but we need a way of showing that it is an estimate.
+            '_time_text': time_text,
+            'timestamp': timestamp,
+        })
+
+        info['author_url'] = urljoin(
+            'https://www.youtube.com',
+            traverse_obj(comment_entity_payload,
+                         ('author',
+                          'channelCommand',
+                          'innertubeCommand',
+                          'browseEndpoint',
+                          'canonicalBaseUrl'),
+                         expected_type=str, get_all=False))
+
+        author_is_uploader = traverse_obj(comment_entity_payload, ('author', 'isCreator'))
```

Comment:
```suggestion
        author_is_uploader = traverse_obj(comment_renderer, ('author', 'isCreator', {bool}))
```

Reply:
This appears to have no effect ```isCreator``` is a string in the set ```{'true', 'false'}```, so while I tested the change, I did not commit it. Looking at "old" ```.info.json``` files confirms this is the original format.

Reply:
I'm getting confused when you say "old `.info.json`": the infojson files created by `--write-comments` or `--write-info-json` would only have the values that yt-dlp sets, not the actual JSON responses from youtube